rubycritic.gemspec

```diff
@@ -35,4 +35,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'minitest', '~> 5.3'
   spec.add_development_dependency 'mocha', '~> 1.1'
   spec.add_development_dependency 'rubocop', '0.37.2'
+  spec.add_development_dependency 'pry-byebug'
```

Comment:
Not really necessary for this PR but I believe we should have this in our dev deps anyway and I am a big believer of the boyscout rule ;)